app/config/config.yml

```diff
@@ -76,6 +76,9 @@ doctrine:
             default:
                 connection: default
                 auto_mapping: false
+                metadata_cache_driver: array
+                query_cache_driver: array
+                result_cache_driver: array
```

Comment:
Yes. That's why I say they are just placeholders.
This values have to be changed to the final user (to apc, redis)... and they need to be placed here